test/elixir-mode-font-test.el

```diff
@@ -139,13 +143,15 @@ end"
     (should (eq (elixir-test-face-at 9) 'elixir-atom-face))
     (should (eq (elixir-test-face-at 10) 'elixir-atom-face))))
 
-(ert-deftest elixir-mode-syntax-table/fontify-interpolation ()
-  :tags '(fontification interpolation syntax-table)
-  (elixir-test-with-temp-buffer
-      "\"#{1 + 2} is 3.\""
-    (should (eq (elixir-test-face-at 1) 'font-lock-string-face))
-    (should (eq (elixir-test-face-at 3) 'font-lock-variable-name-face))
-    (should (eq (elixir-test-face-at 11) 'font-lock-string-face))))
+;; (ert-deftest elixir-mode-syntax-table/fontify-interpolation ()
```

Comment:
On 16.03.2015 18:17, Matt DeBoard wrote:
>> @@ -139,13 +143,15 @@ end"
>>       (should (eq (elixir-test-face-at 9) 'elixir-atom-face))
>>       (should (eq (elixir-test-face-at 10) 'elixir-atom-face))))
>>
>> -(ert-deftest elixir-mode-syntax-table/fontify-interpolation ()
>> -  :tags '(fontification interpolation syntax-table)
>> -  (elixir-test-with-temp-buffer
>> -      "\"#{1 + 2} is 3.\""
>> -    (should (eq (elixir-test-face-at 1) 'font-lock-string-face))
>> -    (should (eq (elixir-test-face-at 3) 'font-lock-variable-name-face))
>> -    (should (eq (elixir-test-face-at 11) 'font-lock-string-face))))
>> +;; (ert-deftest elixir-mode-syntax-table/fontify-interpolation ()
>
> No warehoused code. Delete or uncomment.
>



Hmm, don't understand. Already commented at the issue. These test doesn't succeed currently due to TQS-fix, but is valid.